_includes/social.html

```diff
@@ -91,12 +91,31 @@ <h3>Share</h3>
 
     <a
       href="{{ page.url | prepend: site.baseurl | prepend: site.url }}"
-
       title="Permalink"
-      onclick="plausible('Share: Permalink')"
+      onclick="return share()"
     >
       <img src="{{ '/images/edit-copy-32.svg' | prepend: site.baseurl }}" alt="Permalink" />
     </a>
   </div>
 </div>
 
+<script>
+function share() {
+  plausible('Share: Permalink');
+
+  if (navigator.share) {
+    navigator.share({
+      title: "{{ page.title }}",
+      text: "{{ page.description }}",
+      url: "{{ site.url }}{{ site.baseurl }}{{ page.url }}",
```

Comment:
I'd make this the same as the href attribute and use `prepend`. Just cause I like consistency

Reply:
```suggestion
      url: "{{ page.url | prepend: site.baseurl | prepend: site.url }}",
```